src/renderer/dx/ScreenPixelShader.h

```diff
@@ -4,84 +4,170 @@
 const char screenPixelShaderString[] = "";
 #else
 const char screenPixelShaderString[] = R"(
+// In order to run in KodeLife (great Shader IDE and free with some nagging), define the following:

```

Comment:
Perhaps we like to keep the old effect around as well but was uncertain on how to proceed to keep both.